workload/src/client.cc

```diff
@@ -264,6 +265,12 @@ err_t workloadapi_Client_Close(workloadapi_Client *client)
         delete((SpiffeWorkloadAPI::Stub *) client->stub);
         client->owns_stub = false;
     }
+    for (size_t i = 0, length = arrlen(client->context_list); i < length; i++)
+    {
```

Comment:
Me parece que o code-style não foi aplicado a este arquivo. Favor verificar e aplicar, se for o caso.